### ISSUE TITLE
CSV problem

### ISSUE THREAD

**Issue by mleipold** [ParkerICI/premessa#3]
Hi,

I had some problems with my BC key CSV being allowed by the Debarcoder GUI.

I created the BC key in Excel, then saved as CSV.  I then got:
"Error:  duplicate 'row.names' are not allowed"

My row (BC) names were in the format "Donor_Cells", so "Pat03_2M".  I have 2M, 1M, and 05M samples, for each of 3 donors. 

- see BC key.csv and .xlsx

I have been unable to attach CSV files to this post by Dropping them (only xlsx come over), so all the relevant CSV files that I tried can be found here:
https://drive.google.com/open?id=0Bz9sd8nC272ja1ZCS25FY19aNDg

I tried removing the "_", and it wouldn't work (BC key-B.csv and .xlsx files)

I tried the copy and paste-as-text trick mentioned in the MATLAB normalizer Wiki, and it also didn't work (BC key-1.csv)

So, I thought maybe the names were too long, and I started paring them down.  When I got one to work, I extended the name a bit longer and tried again.  Almost all of them worked (only E, a true duplicate, failed) (files through K).

I then started replacing individual lines with my Donor names, again (starting in L), and each of them worked.....all the way out to S, which to my non-computer eyes is *identical*to* the original BC key.csv that *failed*.

In short:  I don't know why my original CSV file was disallowed by the Debarcoder, nor why BC key-S.csv" *is* allowed, even though it looks identical to "BC key.csv"

Please advise.....

Mike

**Comment by rfinck** [comment]
Hi Mike,

In 071917-CHIM-trial4-BC key.csv, you have two rows in the Mass column
labeled Pat05_2M:


​
I didn't check the other files so don't know where else this occurs.

**Comment by mleipold** [comment]
Dang.  I don't know how I missed that.

I agree that's the problem, and consider this closed.  Thanks.

**Comment by pfgherardini** [comment]
Yep! Thanks @rfinck  for catching that
